eng/pipelines/test-integration-helix.yml

```diff
@@ -0,0 +1,126 @@
+parameters:
+  - name: poolName
+    type: string
+  - name: queueName
+    type: string
+  - name: configuration
+    type: string
+    default: 'Debug'
+  - name: timeout
+    type: number
+  - name: testRuns
+    type: object
+    default:
+      - oop64bit: true
+        oopCoreClr: false
+        lspEditor: false
+        runName: 64
+  
+
+stages:
+- stage: Windows_${{ parameters.configuration }}_Build
+  dependsOn: []
+  jobs:
+  - template: build-windows-job.yml
+    parameters:
+      jobName: Build_Windows_${{ parameters.configuration }}
+      testArtifactName: Transport_Artifacts_Windows_${{ parameters.configuration }}
+      configuration: ${{ parameters.configuration }}
+      poolName: ${{ parameters.poolName }}
+      queueName: ${{ parameters.queueName }}
+      restoreArguments: -msbuildEngine vs
+      buildArguments: -msbuildEngine vs
+
+- stage: ${{ parameters.configuration }}_Integration
+  dependsOn: Windows_${{ parameters.configuration }}_Build
+  variables:
+  - name: XUNIT_LOGS
+    value: $(Build.SourcesDirectory)\artifacts\log\Debug
+  jobs:
+  - ${{ each testParameters in parameters.testRuns }}:
+    - job: ${{ testParameters.runName }}
+      timeoutInMinutes: ${{ parameters.timeout }}
+      pool:
+        name: ${{ parameters.poolName }}
+        demands: ImageOverride -equals ${{ parameters.queueName }}
+      steps:
+      - task: DownloadPipelineArtifact@2
+        displayName: Download Test Payload
+        inputs:
+          artifact: Transport_Artifacts_Windows_${{ parameters.configuration }}
+          path: '$(Build.SourcesDirectory)'
+
+      - task: BatchScript@1
+        displayName: Rehydrate RunTests
+        inputs:
+          filename: ./artifacts/bin/RunTests/${{ parameters.configuration }}/net7.0/rehydrate.cmd
+        env:
+          HELIX_CORRELATION_PAYLOAD: '$(Build.SourcesDirectory)\.duplicate'
+      
+      # This is a temporary step until the actual test run moves to helix (then we would need to rehydrate the tests there instead)
```

Comment:
these will be removed in the next followup where the rehydration happens in the helix script.